packages/lexical-editor/src/nodes/ListNode/WebinyListNode.ts

```diff
@@ -137,11 +143,35 @@ export class WebinyListNode extends ElementNode {
         };
     }
 
+    setDefaultThemeListStyleByTag(tag: string, theme: WebinyTheme) {
```

Comment:
In general, I don't understand why is this needed or what's this fixing. 

I mean, lists are being styled on my end fine, without this logic in place.

https://user-images.githubusercontent.com/5121148/236429980-953367fa-fd6b-422f-9745-e043428210ac.mov




Can you clarify a bit?

Reply:
When you try to copy some lists from other web pages or google docs will not be styled properly.